employees/tests/test_unit_report_model.py

```diff
@@ -156,20 +145,25 @@ def test_report_model_work_hours_field_should_not_be_empty(self):
 
     # PARAM
     def test_report_model_work_hours_field_should_not_accept_value_exceeding_set_digits_number(self):
-        self.field_should_not_accept_input('work_hours', self.WORK_HOURS_DATA_EXCEEDING_DIGITS_NUMBER)
+        input_data = generate_decimal(digits=ReportModelConstants.MAX_DIGITS.value + 1)
+        self.field_should_not_accept_input('work_hours', input_data)
```

Comment:
@kbeker Is this more to your liking?

Reply:
@maciejSamerdak Try to inline it like this: ```self.field_should_not_accept_input('work_hours', generate_decimal(digits=ReportModelConstants.MAX_DIGITS.value + 1))

If it will be longer than 120 character, leave it this way. :)

Reply:
Done